scene/gui/line_edit.h

```diff
@@ -150,6 +157,8 @@ class LineEdit : public Control {
 	virtual bool can_drop_data(const Point2 &p_point, const Variant &p_data) const;
 	virtual void drop_data(const Point2 &p_point, const Variant &p_data);
 
+	virtual Control::CursorShape get_cursor_shape(const Point2 &p_pos) const;
```

Comment:
`LineEdit` inherits `Control`, so you might be able to use `CursorShape` directly without the namespace, no?